examples/webgl_materials_nodes.html

```diff
@@ -1875,7 +1875,6 @@
 						].join( "\n" ) );
 
 						var voronoi = new Nodes.FunctionNode( [
-							// Based off of iq's described here: http://www.iquilezles.org/www/articles/voronoili
```

Comment:
I believe this is https://www.iquilezles.org/www/articles/voronoilines/voronoilines.htm

Reply:
Wasn't sure about that. The formula in the article looks different to what is implemented in the example.

Reply:
557fd10bc596ab72e2bf7b61284a3dd979887874